### DISCUSSION TITLE
Making $SUS availabe for trade on Phantom Wallet

### DISCUSSION THREAD

**Discussion by ghost** [phantom/docs#156] [General]
I'm part of the team at $SUS, a memecoin on Solana.

We'd like to add $SUS to the Phantom Wallet so that users can easily trade it from the wallet interface

As such we'd like to know the next steps to proceed with this

**Comment by adamdelphantom** [comment]
@devissus Thanks for writing in! I would recommend getting the token listed on coin gecko. Would you also mind providing the token address, please?
